app/assets/javascripts/qle.js.erb

```diff
@@ -199,7 +199,7 @@ $(function () {
       get_qle_date();
     } else {
       $('#qle_reason').hide();
-      var errorNoticeAction = "Based on the information you have provided, you are not eligible for this special enrollment period. If you have questions or would like to provide additional information, please contact DC Health Link customer service at (855) 532-5465.";
+      var errorNoticeAction = "Based on the information you have provided, you are not eligible for this special enrollment period. If you have questions or would like to provide additional information, please contact Cover ME customer service at (855) 532-5465.";
```

Comment:
This is a `.js.erb` file so we can probably reference the `#{Settings.site.short_name}` in the `disable_shop_base`.